pkg/repository/crypto/keys.go

```diff
@@ -21,11 +21,26 @@ var (
 	// ErrorKeyUninitialized will be present when key is used before Deserialize called
 	ErrorKeyUninitialized = errors.New("key not initialized, call Deserialize first")
 	// ErrorDeserializeKey means the key format is not valid
-	ErrorDeserializeKey = errors.New("error on deserialize key, check if the key is valid")
+	ErrorDeserializeKey = errors.New("Error on deserialize key, check if the key is valid")
```

Comment:
The error message should be started with the lower case (which is more Golang idiom).